google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/ITInstanceAdminExample.java

```diff
@@ -98,14 +99,14 @@ public void testCreateAndDeleteInstance() throws IOException {
     // Creates an instance.
     String fakeInstance = generateId();
     String fakeCluster = generateId();
```

Comment:
Why are these fake?
Why can't cluster name be deterministic?

Reply:
If a cluster with that name exists in a different instance, AlreadyExists exception will be thrown. That't why I used generateId() method here. Is that okay?